prime-router/metadata/fhir_mapping/hl7/resource/DiagnosticReport.yml

```diff
@@ -159,13 +159,10 @@ issued:
     dateTimeIn: OBR.22
 
 resultsInterpreter:
-  condition: $interpreter NOT_NULL
-  valueOf: resource/Practitioner
+  valueOf: datatypes/CNN/Practitioner
```

Comment:
This is a temporary mapping for the sole purpose of testing the CNN datatype. The intended final mapping is NDR, which contains CNN as a sub datatype.